components/Chips/tests/unit/ChipsTests.m

```diff
@@ -0,0 +1,115 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <XCTest/XCTest.h>
+
+#import "MaterialChips.h"
+
+static inline UIColor *MDCColorFromRGB(uint32_t rgbValue) {
+  return [UIColor colorWithRed:((CGFloat)((rgbValue & 0xFF0000) >> 16)) / 255
+                         green:((CGFloat)((rgbValue & 0x00FF00) >> 8)) / 255
+                          blue:((CGFloat)((rgbValue & 0x0000FF) >> 0)) / 255
+                         alpha:1];
+}
+
+static inline UIColor *MDCColorDarken(UIColor *color, CGFloat percent) {
+  CGFloat hue;
+  CGFloat saturation;
+  CGFloat brightness;
+  CGFloat alpha;
+  [color getHue:&hue saturation:&saturation brightness:&brightness alpha:&alpha];
+
+  brightness = MIN(1, MAX(0, brightness - percent));
+
+  return [UIColor colorWithHue:hue saturation:saturation brightness:brightness alpha:alpha];
+}
+
+static inline UIColor *MDCColorLighten(UIColor *color, CGFloat percent) {
+  return MDCColorDarken(color, -percent);
+}
+
+@interface ChipsNoopTest : XCTestCase
```

Comment:
Did you mean to call this class `ChipsTests` instead?

Reply:
Yep, good catch. I renamed the file and forgot to rename the class. :P